pixeltable/exprs/expr.py

```diff
@@ -329,11 +330,8 @@ def tbl_ids(self) -> set[UUID]:
         return {ref.col.tbl.id for ref in self.subexprs(ColumnRef)} | {ref.tbl.id for ref in self.subexprs(RowidRef)}
 
     @classmethod
-    def list_tbl_ids(cls, expr_list: list[Expr]) -> set[UUID]:
-        ids: set[UUID] = set()
-        for e in expr_list:
-            ids.update(e.tbl_ids())
-        return ids
+    def all_tbl_ids(cls, exprs: Iterable[Expr]) -> set[UUID]:
```

Comment:
I don't like using 'exprs' as a variable, because that's the canonical module name.